crates/common/tedge_config/src/models/file_path.rs

```diff
@@ -13,15 +13,36 @@ pub struct FilePath(PathBuf);
 #[error("FilePath to String conversion failed: {0:?}")]
 pub struct FilePathToStringConversionFailure(std::ffi::OsString);
 
-impl<T> From<T> for FilePath
-where
-    PathBuf: From<T>,
-{
-    fn from(input: T) -> Self {
```

Comment:
to fix the clippy warning here, i needed to add
```rust
impl From<FilePath> for Pathbuf
``` 
but it was not letting me because of this generic impl<T> From<T>.

Reply:
This is why I would ignore this clippy comment.

Reply:
You mean `disable` clippy lint check on that piece of code?

Reply:
No, I mean disabling the clippy warning that is not compatible with this generic implementation. See https://github.com/thin-edge/thin-edge.io/pull/1191/files#r896845061

Reply:
Ok, got it. Thanks.